Tests/UnitTests/FoundationExtensions/DecoderExtensionTests.swift

```diff
@@ -89,14 +89,83 @@ class DecoderExtensionsIgnoreErrorsTests: XCTestCase {
     }
 
     func testIgnoresErrors() throws {
-        let json = "{\"url\": \"not a! valid url@\"}".data(using: .utf8)!
-        let data: Data = try JSONDecoder.default.decode(jsonData: json)
+        let json = "{\"url\": \"not a! valid url@\"}"
+        let data = try Data.decode(json)
 
         expect(data.url).to(beNil())
     }
 
 }
 
+class DecoderExtensionsLossyCollectionTests: XCTestCase {
+
+    private struct Data: Codable, Equatable {
```

Comment:
I don't _think_ this needs to be changed but I was confused at how `Data` was being used thinking that all the `Data` objects in the tests were the `NSData` type of `Data` objects and then not realizing there is a custom `Data` struct here.

I would also name this something like `TestData` or something but that is just kind of a personal nit 🙃

Reply:
Well this isn't just `Data` (as in the global namespace), it's `DecoderExtensionsLossyCollectionTests.Data` (it's namespaced inside the test class).

So in a way, it is like "Test Data". Does that make sense?